code/index.html

```diff
@@ -3,28 +3,142 @@
+          <source src="./videos/lego-ken-burns-1080p.mp4" type="video/mp4" media="screen and (min-device-width:1200px)">
+          </video>
       </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="header-content">
+          <h1>Dream. Build.</h1>
+          <h1>Play.</h1>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+    </header>
+    <article>
+      <div class="text-section">
+        <p>When you want learning to stick, you make it fun. Attend our workshop and discover the benefits of LEGO® play.</p>
+      </div>
+      <div class="block-section">
+        <!-- Accordion -->
+        <div class="accordion">
+          <h2>Benefits of LEGO® play</h2>
+          <div class="accordion-item">
+            <a class>
+              🎈 Free your mind
+            </a>
+            <div class="accordion-content">
+            <p>Discover the powerful moments where your mind is free of
+              all other distractions.</p>
+            </div>  
+          </div>
+          <div class="accordion-item">
+            <a class>
+              💡 Get creative every day
+            </a>
+            <div class="accordion-content">
+              <p>Be as adventurous as you want, experiment, test out and
+                build new ideas.</p>
+            </div>
+          </div>
+          <div class="accordion-item">
+            <a class>
+              🔥 Improve self-discipline
+            </a>
+            <div class="accordion-content">
+              <p>Prove yourself and accomplish the goals you set.</p>
+            </div>
+          </div>
+          <div class="accordion-item">
+            <a class>
+              🚀 Increase spatial awareness
+            </a>
+            <div class="accordion-content">
+              <p>Help keep your critical thinking skills sharp.</p>
+            </div>
+          </div>
+        </div>
+        <!-- Signup form -->
+        <div class="form-container">
+          <form action="http://httpbin.org/anything" method="POST" id="signupform">
+            <h2>Register for a workshop</h2>
+            <ul>
+              <li>
+                <label for "first-name">First name:</label>
+                <input
+                  class="text-input"
+                  type="text"
+                  id="first-name"
+                  name="first-name"
+                  placeholder="Enter first name"
+                />
+              </li>
+              <li>
+                <label for "last-name">Last name:</label>
+                <input
+                  class="text-input"
+                  type="text"
+                  id="last-name"
+                  name="last-name"
+                  placeholder="Enter last name"
+                />
+              </li>
+              <li>
+                <label for "email"><sup>*</sup>Email:</label>
```

Comment:
I like the sup-tag. Suggestion - put the * on the same side here as in terms and conditions in row 126 to make it more unitary.